packages/graph/hash_graph/lib/graph/src/store/query/filter.rs

```diff
@@ -741,134 +748,162 @@ mod tests {
 
     #[test]
     fn for_entity_by_edition_id() {
-        let entity_edition_id = EntityEditionId::new(
+        let _entity_edition_id = EntityEditionId::new(
             EntityId::new(
                 OwnedById::new(AccountId::new(Uuid::new_v4())),
                 EntityUuid::new(Uuid::new_v4()),
             ),
-            EntityVersion::new(Timestamp::default()),
+            EntityRecordId::new(0),
+            EntityVersion::new(
+                DecisionTimespan::from(DateTime::default()..),
+                TransactionTimespan::from(DateTime::from(SystemTime::now())..),
+            ),
         );
 
-        let expected = json! {{
-          "all": [
-            { "equal": [
-              { "path": ["ownedById"] },
-              { "parameter": entity_edition_id.base_id().owned_by_id() }
-            ]},
-            { "equal": [
-              { "path": ["uuid"] },
-              { "parameter": entity_edition_id.base_id().entity_uuid() }
-            ]},
-            { "equal": [
-              { "path": ["version"] },
-              { "parameter": entity_edition_id.version() }
-            ]}
-          ]
-        }};
-
-        test_filter_representation(
-            &Filter::for_entity_by_edition_id(entity_edition_id),
-            &expected,
-        );
+        // TODO: `EntityRecordId` is not exposed in the public API, so we can't
```

Comment:
We need `EditionId` to continue to be queryable for non-breaking changes so I would have expected these tests to check that by using the `EntityVersion`, with a `todo` to check the `RecordId` _when it becomes available in the public API_.

Shall we do that, or is there a reason not to?

Reply:
The predefined filters uses the `RecordId` to query instead of comparing `lower(transaction_time)` the result set is the same as the lower bounds of the time axes are immutable.

I'm currently doing some work on exposing the record id btw, so with that we can enable those tests.

Reply:
Cool, can wait for follow-up then